### ISSUE TITLE
bs 5 update ?

### ISSUE THREAD

**Issue by micker** [kaoz70/grapesjs-blocks-bootstrap4#52]
hello did you plan to update to bootstrap 5 ?
regards

**Comment by kaoz70** [comment]
Yes, its something I have been thinking about for a while but haven't gotten my head around of how to do it yet:

Should I create a new repo?
Should I create a new branch and rename this repo?

Since this repo was originally created for Bootstrap's version 4.

**Comment by micker** [comment]
a new branch ?

**Comment by lane711** [comment]
I have the same question - It would probably make sense to create a new branch instead of a separate repo, at least for initial development. Bootstrap 5 doesn't have as many breaking changes as encountered going from bs3 to 4 so I wouldn't anticipate a massive amount of changes. 

I'm evaluating using GrapeJS in an Nodejs based CMS that I have been working on for a couple+ years : https://github.com/lane711/sonicjs

Grape looks pretty awesome so I'm exited about the possibilities here but definitely need to support Bootstrap 5.

**Comment by lane711** [comment]
Here is a separate fork/branch with Bootstrap 5. I had to make a couple small changes, but seems to work pretty well so far:
https://github.com/lane711/grapesjs-blocks-bootstrap4/tree/bootstrap-5

**Comment by micker** [comment]
cool thanks !
